<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditView</class>
 <widget class="QWidget" name="CreditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>450</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Quicksand</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Credit</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color: rgb(255, 255, 255)</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>541</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
    background-color: rgb(255, 156, 58);
    color: white;
}
QTableWidget {
    background-color: rgb(0, 0, 0);
color: white;
}
</string>
   </property>
   <column>
    <property name="text">
     <string>Месяц</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Сумма платежа</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Платеж по процентам</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>По основному долгу</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Остаток долга</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(66, 66, 66)</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 0px solid gray rgb(255, 209, 0);
   color: white;
   border-radius: 30px;
   background-color: rgb(255, 150, 17);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>198</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand</family>
      <pointsize>17</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
   color: white;
</string>
    </property>
    <property name="text">
     <string>Кредитный калькулятор</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="percent_credit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
   color: white;
</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="year_and_month">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
   color: white;
</string>
    </property>
    <item>
     <property name="text">
      <string>Месяцев</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Лет</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
   color: white;
</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
   color: white;
</string>
    </property>
    <property name="title">
     <string>Тип ежемесячных платежей</string>
    </property>
    <widget class="QRadioButton" name="radioButton_aut">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
   color: white;
</string>
     </property>
     <property name="text">
      <string>Аннуитетные</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_dif">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
   color: white;
</string>
     </property>
     <property name="text">
      <string>Дифференцированные</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
   color: white;
</string>
    </property>
    <property name="text">
     <string>р</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
   color: white;
</string>
    </property>
    <property name="text">
     <string>Сумма кредита</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sum_credit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
   color: white;
</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="time_credit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
   color: white;
</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
   color: white;
</string>
    </property>
    <property name="text">
     <string>Срок кредита</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
   color: white;
</string>
    </property>
    <property name="text">
     <string>Процентная ставка</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>421</x>
      <y>0</y>
      <width>31</width>
      <height>451</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
